### PR TITLE
Adding opa policy to block rollout max unavailability of > than 25%

### PR DESCRIPTION
**Pull request by rwsweeney**
## Description

This policy checks to see if the maxUnavailable value for a rollingdeployment is greater than 25%, and if it is it creates an action item.

## Checklist

* [X] I have updated the CHANGELOG of any plugins updated
* [X] I have updated the version file of any plugins updated
* [X] I have added tests wherever appropriate
* [X] I have labeled this PR with a label per plugin updated

### CODE REVIEW

File: plugins/opa/examples/rollout-strategy/policy.rego [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  not blockedNamespace(input)
+  
+  # These lines check to see if the maxUnavailable field is a percentage, and if so to strip it and converts it to a number for evaluation.
+  maxUnavailable := input.spec.strategy.rollingUpdate.maxUnavailable
+  contains(maxUnavailable, "%")
+  maxUnavailble = replace(maxUnavailable, "%", "")
+  newMaxUnavailable = to_number(maxUnavailble)
+  
+  # If newMaxUnavailable is greater than 25 then create an action item.
+  newMaxUnavailable = 25
```
Comment (rbren) [inline plugins/opa/examples/rollout-strategy/policy.rego:21]: would suggest making this `==` per [these docs](https://www.openpolicyagent.org/docs/latest/policy-language/#equality-assignment-comparison-and-unification)

Actually I'm not sure if `=` works here? I don't really understand how `=` works in rego.

Comment (rbren) [reply plugins/opa/examples/rollout-strategy/policy.rego:21]: Oh actually - reading your comment - I think this should be `newMaxUnavailable > 25`

Comment (rwsweeney) [reply plugins/opa/examples/rollout-strategy/policy.rego]: Ooh, good catch on newMaxUnavailable > 25! Not sure how that ended up as an =. That should now be fixed.

The single equals sign works in my test cluster, and when I add == for line 17 and 18 I get the following errors:
fairwinds:17: rego_unsafe_var_error: var maxUnavailble is unsafe
fairwinds:18: rego_unsafe_var_error: var newMaxUnavailable is unsafe

File: plugins/opa/version.txt [hunk 2]
```diff
@@ -1 +1 @@
-0.3.9
+0.3.91
```
Comment (rbren) [inline plugins/opa/version.txt:1]: I think this should be `0.3.10`

File: plugins/opa/examples/rollout-strategy/helm-deployed.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+parameters:
```
Comment (rbren) [inline plugins/opa/examples/rollout-strategy/helm-deployed.yaml:1]: I think this file should be called `deployments.yaml` or similar

**Review by rbren — COMMENTED** [review]
Thanks Rachel!

**Review by rbren — APPROVED** [review]

**Pull request merged**